content/telegraf/v1/install.md

```diff
@@ -60,64 +38,180 @@ Telegraf uses a host's local time in UTC to assign timestamps to data.
 Use the Network Time Protocol (NTP) to synchronize time between hosts. If hosts' clocks
 aren't synchronized with NTP, the timestamps on the data might be inaccurate.
 
-## Install
+## Download and install Telegraf
+
+<span id="installation"></span>
+
+_**Recommended**_: Before you open and install packages and downloaded files, use SHA
+checksum verification and GPG signature verification to ensure the files are
+intact and authentic.
+
+SHA checksum and GPG signature verification are complementary checks.
+
+_For some Linux platforms, the [installation instructions](#download-and-install-instructions) include steps to verify downloaded packages and binaries._
+
+For more information, see the following:
+
+{{< expand-wrapper >}}
+{{% expand "Verify download integrity using SHA-256" %}}
+
+For each released binary, InfluxData publishes the SHA checksum that
+you can use to verify that the downloaded file is intact and hasn't been corrupted.
+
+To use the SHA checksum to verify the downloaded file, do the following:
+
+1. In the [downloads page](https://www.influxdata.com/downloads),
+   select the **Version** and **Platform** for your download.
+
+   The page displays the unique SHA256 checksum for the file--for example:
+
+   {{< img-hd src="/img/telegraf/downloads-telegraf-linux-amd64.png" alt="Screenshot of InfluxData downloads page showing the Telegraf SHA256 and code snippet" />}}
```

Comment:
I would hesitate to put a screenshot here, just because of maintainability (version changes, downloads page changes, etc.).

Reply:
Thanks. I'll remove it. I asked Edge if they would provide the `.sha256` file as a separate download, which would simplify this a bit.